modules/virt-creating-vm-cloned-pvc-data-volume-template.adoc

```diff
@@ -1,73 +1,75 @@
+You can create a virtual machine (VM) that clones the persistent volume claim (PVC) of an existing VM by using a data volume template. This method creates a data volume whose lifecycle is independent on the original VM.
 
 .Prerequisites
 
 * The VM with the source PVC must be powered down.
+* You must have the `virtctl` command line tool installed.
 
 .Procedure
 
-. Create a `VirtualMachine` manifest as shown in the following example:
+. Create a `VirtualMachine` manifest for your VM and save it as a YAML file, for example:
++
+[source,terminal]
+----
+$ virtctl create vm --name rhel-9-clone --volume-import type:pvc,src:my-project/imported-volume-q5pr9
+----
+
+. Review the `VirtualMachine` manifest for your VM:
 +
 [source,yaml]
 ----
 apiVersion: kubevirt.io/v1
 kind: VirtualMachine
 metadata:
-  labels:
-    kubevirt.io/vm: vm-dv-clone
-  name: vm-dv-clone <1>
+  name: rhel-9-clone # <1>
 spec:
-  runStrategy: Halted
-  template:
-    metadata:
-      labels:
-        kubevirt.io/vm: vm-dv-clone
-    spec:
-      domain:
-        devices:
-          disks:
-          - disk:
-              bus: virtio
-            name: root-disk
-        resources:
-          requests:
-            memory: 64M
-      volumes:
-      - dataVolume:
-          name: favorite-clone
-        name: root-disk
   dataVolumeTemplates:
   - metadata:
-      name: favorite-clone
+      name: imported-volume-h4qn8
     spec:
-      storage:
-        accessModes:
-        - ReadWriteOnce
-        resources:
-          requests:
-            storage: 2Gi
       source:
         pvc:
-          namespace: <source_namespace> <2>
-          name: "<source_pvc>" <3>
+          name: imported-volume-q5pr9 # <2>
+          namespace: my-project # <3>
+      storage:
+        resources: {}
+  instancetype:
+    inferFromVolume: imported-volume-h4qn8 # <4>
+    inferFromVolumeFailurePolicy: Ignore
+  preference:
+    inferFromVolume: imported-volume-h4qn8 # <5>
+    inferFromVolumeFailurePolicy: Ignore
+  runStrategy: Always
+  template:
+    spec:
+      domain:
+        devices: {}
+        memory:
+          guest: 512Mi
+        resources: {}
+      terminationGracePeriodSeconds: 180
+      volumes:
+      - dataVolume:
+          name: imported-volume-h4qn8
+        name: imported-volume-h4qn8
 ----
-<1> Specify the name of the VM.
-<2> Specify the namespace of the source PVC.
-<3> Specify the name of the source PVC.
+<1> The VM name.
+<2> The name of the source PVC.
+<3> The namespace of the source PVC.
+<4> If the PVC source has appropriate labels, the instance type can be inferred from the selected `DataSource` object.
+<5> If the PVC source has appropriate labels, the preference can be inferred from the selected `DataSource` object.
 
 . Create the virtual machine with the PVC-cloned data volume:
 +
 [source,terminal]
 ----
-$ oc create -f <vm-clone-datavolumetemplate>.yaml
+$ oc create -f <vm_manifest_file>.yaml
```

Comment:
I know it was already like this, but I was wondering whether the first step actually saves the manifest as a file already for you, or if it just outputs the YAML to the screen and then you have to copy/paste/save it as a file? 

My reason for asking is 1). it could help to clarify this / maybe it's missing a step, and 2). could we actually show the above YAML example as a particular example file name, and then just use that here?

(Same comment for the other modules, if anything does happen to get updated)